src/compiler/checker.ts

```diff
@@ -5451,6 +5453,11 @@ namespace ts {
                                 localErrors = false;
                             }
                         }
+                        if (localErrors) {

```

Comment:
Can you add documentation to `localErrors` where it's defined? Also, give it a better name because I don't think it accurately reflects the functionality.